src/Splitters/HtmlSplitter.php

```diff
@@ -13,6 +13,7 @@
 
 namespace Algolia\ScoutExtended\Splitters;
 
+use DOMXPath;
```

Comment:
No need to add an extension to `composer.json`?

Reply:
No, it's part of ext-dom 
https://www.php.net/manual/fr/intro.dom.php